GitUI/Infrastructure/Telemetry/DiagnosticsClient.cs

```diff
@@ -1,5 +1,5 @@
 ﻿// The original idea and the implementation are borrowed from  https://github.com/NuGetPackageExplorer/NuGetPackageExplorer
-// Credits to Oren Novotny
+// Credits to Claire Novotny
```

Comment:
Worth changing to a GitHub link to the username (IIRC "onovotny")

Reply:
The GitHub link looks ok to me. Am I missing something?